package.json

```diff
@@ -17,14 +17,14 @@
     "lint:js:changed": "./bin/lint-changed-js",
     "problems": "tsc -p tsconfig.json --noEmit --pretty false",
     "start": "yarn workspace ember-data start",
-    "test": "yarn workspace ember-data test",
+    "test": "lerna run --stream --no-private --ignore='@ember-data/{private-build-infra,canary-features,debug}' test",
```

Comment:
`lerna run` runs the specified command (in this case `test`) in all packages that have the command in scripts in their package.json.

`--stream` tells it to pipe the output to stdout right away vs waiting for the command to finish, this is useful for it to not seem like it's hanging / potentially cause CI to timeout.

`--no-private` is a filter telling it to not run this command for any package marked private in package.json.

`--ignore` tells it to additionally ignore the command in these packages if found. We are ignoring these three because if they do ever have more tests of their own they are likely to be node tests vs tests of addon behavior.